config/demo

```diff
@@ -2,9 +2,9 @@ include Makefile
 
 export instanceid = ui
 export base_url = /
-export api_url = https://api.demov6.camptocamp.org
-export api_url_internal = http://api
-export api_url_host = api
+export api_url = http://api.demov6.camptocamp.org
+export api_url_internal = http://api.demov6.camptocamp.org
```

Comment:
Do we really need to change ``api_url_internal``? The internal URL is the one used by the UI server to connect to the API. If both instances are on the same server (or network) it could be "localhost" (or whatever local host), avoiding to make an request with the external public URL.

The other changes make sense but:
* they are not related to the header color => it would have been better to make a distinct commit (if not PR) for this (only do things related to the commit title)
* the demo config is no longer used I think, since we use docker images to deploy the demo and prod instances (settings are then retrieved from docker compose configs). Perhaps we could simply remove this file :)